src/FakeItEasy/Core/DefaultExceptionThrower.cs

```diff
@@ -78,7 +78,7 @@ private static void AppendNonTriedConstructors(IEnumerable<ResolvedConstructor>
 
                 message
                     .AppendLine()
-                    .AppendIndented("      ", "Types marked with * could not be resolved, register them in the current\r\nIFakeObjectContainer to enable these constructors.")
+                    .AppendIndented("      ", "Types marked with * could not be resolved, register them in the current\r\nIDummyValueCreationSession to enable these constructors.")

```

Comment:
This does hit the mark in removing a reference to `IFakeObjectContainer`, but it can probably be improved.
`IDummyValueCreationSession` is internal, so an end user seeing this message will not know what to do about it.
The only thing they would know about creating Dummies comes from the [How are Dummies made?](http://fakeiteasy.readthedocs.io/en/stable/dummies/#how-are-the-dummies-made) topic (which points to [Custom Dummy Creation](http://fakeiteasy.readthedocs.io/en/stable/custom-dummy-creation/)).

So I think we might be better off suggesting the users provide a Dummy Factory for the indicated types.